File: test/samples/sample.rb
def classify(a, b)
  if a > 0 && b > 0
    1
  elsif a < 0
    2
  else
    3
  end
end

def loop(n, m)
  total = 0
  for i in 0...n
    while total < m
      if total == 7
        total = total + 2
      end
      total = total + 1
    end
  end
  total
end

File: test/samples/sample.py
def classify(a, b):
    if a > 0 and b > 0:
        return 1
    elif a < 0:
        return 2
    else:
        return 3


def loop(n, m):
    total = 0
    for i in range(n):
        while total < m:
            if total == 7:
                total = total + 2
            total = total + 1
    return total
